### ISSUE TITLE
508-defect-4 [COGNITION]: Telephone number on the Review your appointment details screen should be formatted for content and semantics

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#24172]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Details

The user's telephone number should be formatted to match our [design standard for telephone numbers](https://design.va.gov/content-style-guide/dates-and-numbers). This will make it easier for users to review and confirm their phone number is correct.

## Acceptance Criteria

- [ ] Phone number is formatted correctly using our preferred 123-456-7890 style
- [ ] Phone number announces to screen readers as "One two three." "Four five six." instead of longer numbers like "one-hundred twenty-three".
- [ ] Phone number should use the [Telephone component](https://design.va.gov/storybook/?path=/docs/components-telephone--default) OR custom markup that uses an `aria-label` to read phone numbers correctly. It is encouraged to test this with as many screen reader devices as possible.
- [ ] 0 axe-core violations

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/review

## Screenshots or Trace Logs

<img width="1040" alt="Screen Shot 2021-05-04 at 9 31 51 AM" src="https://user-images.githubusercontent.com/934879/117025020-233f9580-acc0-11eb-8e4e-c77189caf3f7.png">

**Comment by laurenernest** [comment]
@1Copenut The Design System documentation says to use hyphens, not parentheses so want to confirm which one we should use.
![Screen Shot 2021-05-04 at 1.56.19 PM.jpg](https://images.zenhubusercontent.com/5f7dd2dad6b9a9301bbab94e/5c2e2630-8e1d-4f42-8d88-c6ff9cb936bd)

**Comment by 1Copenut** [comment]
@laurenernest Good call. I was moving too quickly when I wrote the ticket. I updated the success criteria to reflect this hyphen delineated phone number format.

**Comment by jbalboni** [comment]
@laurenernest I didn't realize there was a display only option, we can definitely switch to that.

The reason the dashes aren't displaying there is because the Telephone component makes an assumption that if the number starts with a 1, that's a country code, since US area codes don't start with 1. So it gets confused and thinks you gave it a 9 digit number, which it can't format. The typical validation for telephone fields in our forms just checks length, so we don't flag it as invalid.

**Comment by jbalboni** [comment]
@laurenernest I put up another PR to address making the numbers not clickable and also updating our phone validation to catch invalid 10 digit phone numbers.

**Comment by jbalboni** [comment]
The updated version of this should be on staging later this afternoon.

**Comment by laurenernest** [comment]
@1Copenut ready for your review
